docs/guides/cluster-to-advanced-cluster-migration-guide.md

```diff
@@ -0,0 +1,175 @@
+2. Have existing `mongodbatlas_cluster` resource(s) and don't want to use an external script for migrating.
+
+#### Steps
+
+1. Find the import IDs of the clusters you want to migrate: `{PROJECT_ID}-{CLUSTER_NAME}`, e.g., `664619d870c247237f4b86a6-legacy-cluster`
+2. Add an import block per cluster to one of your `.tf` files:
+  ```terraform
+  import {
+    to = mongodbatlas_advanced_cluster.this
+    id = "664619d870c247237f4b86a6-legacy-cluster" # from step 1
+  }
+  ```
+3. Run `terraform plan -generate-config-out=cluster.tf`, should generate a `cluster.tf` file and display a message similar to `Plan: 1 to import, 0 to add, 0 to change, 0 to destroy`:
+  ```terraform
+  resource "mongodbatlas_advanced_cluster" "this" {
+    # ... most attributes are removed for readability of this guide
+    # ....
+    backup_enabled                       = true
+    cluster_type                         = "REPLICASET"
+    disk_size_gb                         = 10
+    name                                 = "legacy-cluster"
+    project_id                           = "664619d870c247237f4b86a6"
+    state_name                           = "IDLE"
+    termination_protection_enabled       = false
+    version_release_system               = "LTS"
+
+    advanced_configuration {
+      default_read_concern                 = null
+      default_write_concern                = null
+      fail_index_key_too_long              = false
+      javascript_enabled                   = true
+      minimum_enabled_tls_protocol         = "TLS1_2"
+      no_table_scan                        = false
+      oplog_min_retention_hours            = 0
+      oplog_size_mb                        = 0
+      sample_refresh_interval_bi_connector = 0
+      sample_size_bi_connector             = 0
+      transaction_lifetime_limit_seconds   = 0
+    }
+
+    replication_specs {
+      container_id = {
+        "AWS:US_EAST_1" = "669644ae01bf814e3d25b963"
+      }
+      id         = "66978026668b7619f6f48cf2"
+      zone_name  = "ZoneName managed by Terraform"
+
+      region_configs {
+        priority              = 7
+        provider_name         = "AWS"
+        region_name           = "US_EAST_1"
+
+        auto_scaling {
+          compute_enabled            = false
+          compute_max_instance_size  = null
+          compute_min_instance_size  = null
+          compute_scale_down_enabled = false
+          disk_gb_enabled            = false
+        }
+
+        electable_specs {
+          disk_iops       = 3000
+          ebs_volume_type = null
+          instance_size   = "M10"
+          node_count      = 3
+        }
+        analytics_specs {
+          disk_iops       = 3000
+          ebs_volume_type = null
+          instance_size   = "M10"
+          node_count      = 1
+        }
+      }
+    }
+  }
+  ```
+4. Run a `terraform apply`, you should expect to see the resource imported.
```

Comment:
```suggestion
4. Run `terraform apply`. You should see the resource imported.
```

Reply:
Thanks changed in 288f7f76a58ef30cf833e0adbb1e9543b719d138 and 3a97436556d18d9312d3bd84a9e5cbe6e6e6d006